lib/preload.js

```diff
@@ -91,7 +103,38 @@ const createContainerDisposer = (docker, options) => {
   })
 }
 
-const runContainer = (docker, container) => {
+/**
+ * @description
+ * preload.py is running inside the outer Docker container.
+ * We wait until it outputs something on stdout, that means that the inner
+ * Docker is ready.
+ * Then we pull the build and signal the Python script that we're done by
+ * writing on its stdin.
+ *
+ * @param {Stream} stdout - Python script's stdout
+ * @param {Stream} stdin - Python script's stdin
+ * @param {Number} dockerPort - The port on which the inner dockerd is listening
+ * @param {String} imageId - The image to pull
+ */
+const pullBuild = (stdout, stdin, dockerPort, imageId) => {
+  stdout.on('data', (data) => {
+    const innerDockerProgress = new dockerProgress.DockerProgress({
+      Promise,
+      host: '0.0.0.0',
```

Comment:
Should the host be configurable here?

Reply:
I don't think so.
What would you want to use instead of `0.0.0.0`?

Reply:
Not sure, was just thinking that some consumer might possibly fancy having the option to change it, but I guess that's a bit overkill for now.